clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -151,12 +142,11 @@ describe("SettingsService evaluate variables", () => {
 
 test("getWorkspaceFoldersPath return an array of paths", () => {
   (vscode.workspace.workspaceFolders as any) = [
-    { uri: { path: "/ws-vscode" } } as any,
+    { uri: { path: makePath("/ws-vscode") } } as any,
   ];
   const paths = SettingsUtils.getWorkspaceFoldersPath();
-  expect(paths).toStrictEqual(["/ws-vscode"]);
+  expect(paths).toStrictEqual([makefsPath("/ws-vscode")]);
```

Comment:
getWorkspaceFoldersPath without an argument returns URI path, so revert this part.